### DISCUSSION TITLE
My BufNewFile autocommand doesn't do anything, how do I fix this?

### DISCUSSION THREAD

**Discussion by subhanepix** [LazyVim/LazyVim#3998] [Q&A]
Before I start, let me say that I'm very new to vim, nvim, the whole thing. I've also never written in lua and I generally barely know what I'm doing here. 

Now, all I'm trying to do is make some starting code for a new html file, (later on I want to add other languages so if someone could also help me out with that I'd appreciate it, although I'd imagine I would just do the same thing again with .otherlanguage instead of .html)

 In my autocmds.lua file, I have this code

 vim.api.nvim_create_autocmd("BufNewFile", {
  pattern = "*.html",
  group = vim.api.nvim_create_augroup("skelly", { clear = true }),
  command = "0r ~/.config/nvim/templates/html.html"
})

and for some reason it just doesn't do anything when I make a new html file. 
If it matters, I'm using WSL, and I'm making the new files with neotree by going into the directory and hitting "a" and typing in the name of my new file, with .html at the end. 

I'm sure I'm just doing something slightly wrong but I would really appreciate some guidance as I've been googling for a while now to no avail.

**Comment by dpetka2001** [comment]
I did something similar in the past but don't use it any more. Here's the code for the relevant part in `neo-tree` spec [from my dotfiles](https://github.com/dpetka2001/dotfiles/blob/5ca09b2d1d12c2bd33d9616cec2ab6ca95d0f5f6/dot_config/nvim/lua/plugins/editor.lua#L49-L64). I took advantage of `neo-tree`'s own event handlers. Surely you can modify it to suit your needs.

**Reply by subhanepix** [reply]
I'm gonna give it a shot, I really wanted to make it work through the autocmds so it can still work even if I don't use neo-tree. 

I'm also just genuinely curious why my autocmd just isn't working.
